src/app/modules/operations/uploading-csv/uploading-csv.component.ts

```diff
@@ -44,11 +44,13 @@ export class UploadingCsvComponent implements OnInit {
       name: [""],
       fileName: [""]
     });
-    this.route.parent.queryParams.subscribe(params => {
-      this.programId = params['programId'];
-      this.assessmentId = params['assessmentId']
+    // this.route.parent.queryParams.subscribe(params => {
+    //   this.programId = params['programId'];
+    //   this.assessmentId = params['assessmentId']
 
-    });
+    // });
+    this.programId = localStorage.getItem('programId');
```

Comment:
Use 2 keys, selectedProgram and selectedAssessment which holds the complete object of each so that we can use it for breadcrumbs later on